libraries/AP_Logger/AP_Logger.cpp

```diff
@@ -102,6 +102,14 @@ const AP_Param::GroupInfo AP_Logger::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("_FILE_MB_FREE",  7, AP_Logger, _params.min_MB_free, 500),
 
+    // @Param: _FILE_MB_ROT
+    // @DisplayName: Start a new log file when the current log file reaches this size/
```

Comment:
extra /

Reply:
This has now been removed - @tridge you were right it leads to control issues in flight and seems too risky at this stage